cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```

Comment:
I am not 100% sure what this file will be used for. Should we keep it or remove it?

Reply:
keep it.